File: java/src/com/company/BankAccount.java
package com.company;

/**
 * Class for bank accounts actions
 */
public class BankAccount {
    private int availableAmount;

    BankAccount(int availableAmount) {
        this.availableAmount = availableAmount;
    }

    /**
     * Function to deposit money on the bank account
     * @param amount int
     * @return boolean
     */
    boolean deposit(int amount) {
        if (amount % 10 == 0 && amount > 0) {
            availableAmount += amount;
            System.out.println(amount + "€ added to your account.");
            return true;
        }
        else {
            System.out.println("You only can deposit an amount of a multiple of 10");
            return false;
        }
    }

    /**
     * Function to withdraw money from the bank account
     * @param amount int
     * @return boolean
     */
    boolean withdraw(int amount) {
        if (amount % 10 != 0 || amount <= 0) {
            System.out.println("You only can withdraw an amount of a multiple of 10");
            return false;
        }
        else if (availableAmount >= amount) {
            availableAmount -= amount;
            System.out.println(amount + "€ withdraw from your account.");
            return true;
        }
        else {
            System.out.println("You don't have enough money");
            return false;
        }
    }

    /**
     * Function to check the available amount on the bank account
     * @return int
     */
    int getAvailableAmount() {
        return availableAmount;
    }
}

File: csharp/BankATM/BankAccount.cs
using System;

namespace BankATM
{
    public class BankAccount
    {
        private int availableAmount;

        public BankAccount(int availableAmount)
        {
            this.availableAmount = availableAmount;
        }

        public bool deposit(int amount)
        {
            if (amount % 10 == 0 && amount > 0)
            {
                availableAmount += amount;
                Console.WriteLine("{0}€ added to your account.", amount);
                return true;
            }
            else
            {
                Console.WriteLine("You only can deposit an amount of a multiple of 10");
                return false;
            }
        }

        public bool withdraw(int amount)
        {
            if (amount % 10 != 0 || amount <= 0)
            {
                Console.WriteLine("You only can withdraw an amount of a multiple of 10");
                return false;
            }
            else if (availableAmount >= amount)
            {
                availableAmount -= amount;
                Console.WriteLine("{0} € withdraw from your account.", amount);
                return true;
            }
            else
            {
                Console.WriteLine("You don't have enough money");
                return false;
            }
        }

        public int getAvailableAmount()
        {
            return availableAmount;
        }
    }
}
